umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md

```diff
@@ -55,68 +53,65 @@ Benthos UMH offers some benefits, including:
 ### With the Management Console
 
 The easiest way to use Benthos UMH is to deploy it directly from the Management
-Console
+Console.
 
-{{% notice warning %}}
-Currently, only OPC-UA data sources can be configured from the Management
-Console. To use other data sources, you must deploy Benthos UMH in standalone
-mode or use [Node-RED](/docs/features/connectivity/node-red/).
-{{% /notice %}}
+After adding your network device or service, you can initialize the protocol
+converter. Simply click on the **Play** button next to the network device/service
+at the **Protocol Converters** tab.
+From there, you'll have two options to choose from when configuring the
+protocol converter:
 
-You first have to add a new connection to your OPC-UA server from the
-**Connection Management** tab.
+- **Universal Protocol Converter**: Opt for this choice if you need to configure
+  protocol converters for various supported protocols other than OPC-UA. This option
+  will prompt you to define the Benthos input and processor configuration in YAML format.
 
-![Add Connection](/images/features/data-connectivity-benthos/connection-management.png?width=80%)
+- **OPC-UA**: Select this option if you specifically need to configure
+  OPC-UA protocol converters. You'll be asked to define OPC-UA nodes in YAML format,
+  detailing the nodes you want to stream from the OPC-UA server.
 
-Afterwards, you can initialize the connection by pressing the **Play** button
-next to the connection. Select the correct authentication method and enter the
-OPC-UA nodes you want to stream.
+For OPC-UA, ensure your YAML configuration follows the format below:
 
-Currently, the only method supported for configuring OPC-UA nodes is by
-specifying a YAML file. You can find an example file below:
-
-``` yaml
+```yaml
 nodes:
-    - opcuaID: ns=2;s=Pressure
-      enterprise: pharma-genix
-      site: aachen
-      area: packaging
-      line: packaging_1
-      workcell: blister
-      originID: PLC13
-      tagName: machineState
-      schema: _historian
+  - opcuaID: ns=2;s=Pressure
+    enterprise: pharma-genix
+    site: aachen
+    area: packaging
+    line: packaging_1
+    workcell: blister
+    originID: PLC13
+    tagName: machineState
+    schema: _historian
 ```
 
-Mandatory fields are `opcuaID`, `enterprise`, `tagName` and `schema`. `opcuaID`
+Required fields are `opcuaID`, `enterprise`, `tagName` and `schema`. `opcuaID`
 is the NodeID in OPC-UA and can also be a folder (see [README](https://github.com/united-manufacturing-hub/benthos-umh?tab=readme-ov-file#node-ids)
 for more information). The remaining components are components of the resulting
 topic / ISA-95 structure (see also our [datamodel](/docs/datamodel)). By default,
-the schema will always be in _historian, and tagName is the keyname.
+the schema will always be in \_historian, and tagName is the keyname.
 
 ### Standalone
```

Comment:
Do we need the standalone mode? With our universal protocol converter one can do everything protocol converter related than with the standalone version right?

Reply:
Theoretically not, I left it in if for some reason a user has some very specific usecase where he prefers to deploy his own.